templates/default/snapraid.conf.erb

```diff
@@ -6,11 +9,11 @@
 # Default configuration borrowed from: http://zackreed.me/articles/72-snapraid-on-ubuntu-12-04
 
 <% node['snapraid']['parity_disks'].each_with_index  do |parity, index| %>
-<%=index > 0 ? "#{index+1}-" : ''%>parity <%= parity %>/snapraid.parity
+<%=index > 0 ? "#{index+1}-" : ''%>parity <%= parity + path_separator%>snapraid.parity
```

Comment:
That's what I tried originally, but it always uses forward slash as a separator, which doesn't work with snapraid on Windows!